spec.md

```diff
@@ -72,10 +72,10 @@ This specification defines the following terms:
   and presented to end users of a Platform.
 
 - *Platform*: The software that will manage the cloud environment into which
-  Applications and Service Brokers are provisioned.  Users will not directly
-  provision Services from Service Brokers, rather they will ask the Platform
-  (ie. their cloud provider) to manage Services and interact with the
-  Service Brokers for them.
+  Applications are provisioned and Service Brokers are registered. Users will
+  not directly provision Services from Service Brokers, rather they will ask
+  the Platform (ie. their cloud provider) to manage Services and interact with
```

Comment:
I think this would be a good opportunity to remove the `(ie. their cloud provide)` text. This isn't generally applicable so we should probably just remove it?

Reply:
+1 since we normally just say "Platform" anyway - no real value in introducing yet another term

Reply:
Done.